vscode/src/commands.ts

```diff
@@ -389,6 +392,57 @@ const commandsMap: (state: ExtensionState) => {
     "konveyor.diffView.applySelection": applyBlock,
     "konveyor.diffView.applySelectionInline": applyBlock,
     "konveyor.partialAnalysis": async (filePaths: string[]) => runPartialAnalysis(state, filePaths),
+    "konveyor.configureGetSolutionParams": async () => {
```

Comment:
Just a comment ---

I'm not a fan of the input boxes like this.  I'd rather have a configuration item link to settings.json directly and let code assist handle this kind of base level value.

But that does seem like a future thing to overhaul how the configurations work...